website/i18n/en.json

```diff
@@ -48,6 +48,9 @@
       "contributors/updating-website": {
         "title": "Contributing to the website"
       },
+      "editors/atom": {
```

Comment:
Any idea why this is popping back up again and why the lsp-extension down blow are being removed? This seems like the old stuff before we added the lsp-extensions section and removed the atom page.

Reply:
I think that gets added automatically :thinking: I will check it out tomorrow and finished the release then.

Reply:
I removed it for now and will take a look next week.